secretmanager/snippets/regional_samples/snippets_test.py

```diff
@@ -286,6 +292,28 @@ def test_get_regional_secret_version(
     assert secret_id in version.name
     assert version_id in version.name
 
+def test_iam_grant_access_with_regional_secret(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    regional_secret: Tuple[str, str, str, str],
```

Comment:
This is not a good practice. Kindly find a better alternative.

```
    regional_secret: Tuple[str, str, str, str],
....
....
    project_id, location_id, secret_id, _ = regional_secret
```

Reply:
Can you suggest an alternative? 
The only alternative i can think of is to decompose the list into each variable which will make the code messy. 
Also, path of the secret is project/{project_id}/location/{location_id}/secret/{secret_id}.

Reply:
Sure. A few suggestions on your overall code

1. Generally following CLI code is not encouraged. Your code can be simpler.
```
    parser = argparse.ArgumentParser(
        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
    )
    parser.add_argument("project_id", help="id of the GCP project")
    parser.add_argument("location_id", help="id of location where secret is stored")
```

2. Do decompose the variables. Messy code is better complicated.

    (Better alternative) In test files, `project_id` & `location` could be global variables. Ex: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/generative_ai/context_caching/context_caching_test.py#L26-L28

Reply:
I have fixed 2. Please check.

For 1, it is not included in the samples, it is just there for someone, if they want to run a single sample.